js/main.js

```diff
@@ -0,0 +1,85 @@
+/* Here's what the pine tree should look like when you specify a height of 7, and use the asterisk character.
+
+01        *
+03       ***
+05      *****
+07     *******
+09    *********
+11   ***********
+13  *************
+
+*/
+
+var pineTree = {
+  height: 0,
+  species: "null"
+};
+
+var inputs = document.getElementsByTagName("input");
+var treeHeight = inputs[0];
+var treeSpecies = inputs[1];
+
+document.getElementsByTagName("input")[0].addEventListener("keydown", function(){
+  if(event.keyCode == 13){
+    event.preventDefault();
+    checkValidRun();
+  };
+});
+document.getElementsByTagName("input")[1].addEventListener("keydown", function(){
+  if(event.keyCode == 13){
+    event.preventDefault();
+    checkValidRun();
+  };
+});
+
+document.getElementsByTagName("button")[0].addEventListener("click", function(){
+  checkValidRun();
+});
+
+function checkValidRun(){
+  if(!treeHeight.value || !treeSpecies.value){
+    alert("You must enter both values!");
+  } else if (treeSpecies.value && treeHeight.value) {
+    pineTree.height = treeHeight.value;
+    pineTree.species = treeSpecies.value;
+    growTree(pineTree);
+  }
+};
+
+
+// Create a `tree` function that should build a pine tree out of a character in the Chrome dev tools console. 
+
+function growTree(treeObject){
+  for(let i = 1; i <= treeObject.height; i++){
+    drawOneLine(treeObject.height, i);
+  };
+};
+
+function drawOneLine(totalHeight, rowWereOn){
```

Comment:
Since this function is only being called from one location in your code, then there's no need for the logic to be put into a function.